tournamentmanager/tournamentmanager.py

```diff
@@ -232,6 +236,15 @@ async def tournamentset_settings(self, ctx: commands.Context):
             )
         )
         await ctx.send(embed=embed)
+    @tournamentset.command(name="checkintime")
```

Comment:
Manque un saut à la ligne.

```suggestion

    @tournamentset.command(name="checkintime")
```